test/test_other.py

```diff
@@ -10456,6 +10456,8 @@ def test_minimal_runtime_code_size(self, test_name, js, compare_js_output=False)
                            test_file('minimal_webgl/webgl.c'),
                            '--js-library', test_file('minimal_webgl/library_js.js'),
                            '-lwebgl.js',
+                           '--closure=1',
+                           '-sSTRICT',
```

Comment:
Maybe leave this test as is, since its a sensitive code size test

Reply:
Reverted